packages/kbn-dev-utils/src/proc_runner/proc.js

```diff
@@ -50,14 +50,16 @@ export function createProc(name, { cmd, args, cwd, env, stdin }) {
     }
   }
 
-  const childProcess = spawn(cmd, args, {
+  const childProcess = execa(cmd, args, {
     cwd,
     env,
-    stdio: [stdin ? 'pipe' : 'ignore', 'pipe', 'pipe'],
+    stdio: ['pipe', 'pipe', 'pipe'],
   });
 
   if (stdin) {
```

Comment:
Couldn't the if clause be removed here just leaving `childProcess.stdin.end(stdin, 'utf8');`